Shared/css/style.css

```diff
@@ -7737,6 +7736,82 @@ textarea#mrkdwntxt {
   margin-right: 2px;
 }
 
+.form-popup {
+  text-align: center;
+  display: none;
+  padding: 5px;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  border: 3px solid #f1f1f1;
+  z-index: 9;
+  background-color: #fff; /* white background */
+  height:260px;
+  width:460px;
```

Comment:
Consider using dynamic sizing, this could potentially look strange on devices with different screen sizes.